tests/UKHO.ERPFacade.API.UnitTests/ERPTestData/S100TestData/CancellationAndReplacement.json

```
@@ -84,8 +86,9 @@
                 "providerName": "ICE",
                 "size": "medium",
                 "producingAgency": "GB",
+                "producerUnitName": "US5NY6HV",
                 "status": {
-                    "statusName": "New Edition",
+                    "statusName": "NewEdition",
```

```suggestion
                    "statusName": "newDataset",
```
Although NewEdition will work fine, but technically it should be "newDataset" when edition is 1 and update is 0

---

Changes done as per suggested.